src/body.js

```diff
@@ -302,7 +320,7 @@ export const extractContentType = (body, request) => {
 		return null;
 	}
 
-	if (isFormData(body)) {
+	if (body instanceof FormData) {
```

Comment:
So, with that you will drop support for `formdata-node`? It seem that `formdata-polyfill/esm.min.js` does not recognize `formdata-node` as FormData instance.

<details>
<summary>Details</summary>

![image](https://user-images.githubusercontent.com/7884558/135340796-5ce40e81-6bf8-423b-ae5e-66a00c1682a3.png)

</details>

This change will break compatibility for those who already use `formdata-node` with `node-fetch` v3.

Reply:
Hmm, i thought i formdata-polyfill would use the symbol.hasInstance to accommodate for this.
maybe need to revert 405bcc0 after all

Reply:
Works now

Reply:
Re-added a test case for formdata-node